pulp_ostree/app/tasks/synchronizing.py

```diff
@@ -100,6 +100,11 @@ async def run(self):
                 ref_relative_path = os.path.join("refs/heads/", name)
                 local_ref_path = os.path.join(self.repo_path, ref_relative_path)
 
+                dirname = os.path.dirname(local_ref_path)
+                if not os.path.exists(dirname):
+                    # create a directory to prevent IOError
+                    os.makedirs(dirname)
```

Comment:
```suggestion
                # create a directory to prevent IOError
                os.makedirs(dirname, exist_ok=True)
```
If i remember correctly, the if is builtin.